galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/GalasaResourceType.java

```diff
@@ -5,28 +5,29 @@
  */
 package dev.galasa.framework.api.common.resources;
  
- public enum GalasaResourceType{
-        GALASAPROPERTY("GalasaProperty"),
-        ;
-        private String value;
+public enum GalasaResourceType {
+    GALASA_PROPERTY("GalasaProperty"),
```

Comment:
I wonder if we should be using an OSGi plug-point to add resources, where each resource registers a class with the OSGi service. What do you think?
It would make it more modular, and we could add other resources using different bundles ? Just an idea putting it out there.

Reply:
I like that idea, especially with the improvement to modularity - it would also serve as a good opportunity to become more familiar with the OSGi framework